api/app/signals/apps/signals/models/history.py

```diff
@@ -28,20 +28,21 @@ def delete(self, *args, **kwargs):
     def get_action(self):
         """Generate text for the action field that can serve as title in UI."""
         if self.what == 'UPDATE_STATUS':
-            return 'Update status naar: {}'.format(
-                dict(STATUS_CHOICES).get(self.extra, 'Onbekend'))
+            return f'Update status naar: {dict(STATUS_CHOICES).get(self.extra, "Onbekend")}'
         elif self.what == 'UPDATE_PRIORITY':
             # SIG-1727 ad-hoc translation, must match signals.Priority model!
             translated = {'high': 'Hoog', 'normal': 'Normaal', 'low': 'Laag'}.get(self.extra, 'Onbekend')
-            return 'Prioriteit update naar: {}'.format(translated)
+            return f'Prioriteit update naar: {translated}'
         elif self.what == 'UPDATE_CATEGORY_ASSIGNMENT':
-            return 'Categorie gewijzigd naar: {}'.format(self.extra)
+            return f'Categorie gewijzigd naar: {self.extra}'
         elif self.what == 'CREATE_NOTE' or self.what == 'UPDATE_LOCATION':
             return self.extra
         elif self.what == 'RECEIVE_FEEDBACK':
             return 'Feedback van melder ontvangen'
         elif self.what == 'UPDATE_TYPE_ASSIGNMENT':
             return f'Type update naar: {_history_translated_action(self.extra)}'
+        elif self.what == 'UPDATE_DIRECTING_DEPARTMENTS_ASSIGNMENT':
```

Comment:
Remove "_ASSIGNMENT" for consistency with the /signals endpoint.

Reply:
Will fix that in another PR